book/history.md

```diff
@@ -20,12 +20,14 @@ other predecessors to HTML. (Except in this footnote!)
 The Memex concept
 =================
 
-<style>
-	figure { text-align: center; }
```

Comment:
Why remove the centering?

Reply:
It is just moved to the general book stylesheet.